app/pages/summary/contextCreator.js

```diff
@@ -224,6 +224,12 @@ const generateRecurringCostDetailsTable = ({
       dataTestId: 'item-name',
     }));
 
+    columns.push(({
+      classes: recurringCostTable.cellInfo.serviceItemId.classes,
+      data: item.serviceItemId,
+      dataTestId: 'service-item-id',
```

Comment:
`serviceInstanceId` and `service-instance-id` to be consistent with the business jargon?